JavaScript/examples/a-prod.js

```diff
@@ -7,6 +7,7 @@ const emitter = () => {
     on: (name, f, timeout = 0) => {
       const event = events.get(name);
       if (!event) events.set(name, [f]);
+      else events.set(name, [...event, f]);
```

Comment:
Yes, here is a bug but better we will avoid array spread and reinstantiation `[...event, f]`
```js
if (event) event.push(f);
else events.set(name, [f]);
```